In [1]:
import numpy as np
import pandas as pd
from scipy import stats
import matplotlib.pyplot as plt

EXERCISE 1. Answer the following questions by looking at the distribution table or coding with Python.

1. A salesperson has found that the probability of a sale on a single contact is approximately .3. If the salesperson contacts 10 prospects, what is the approximate probability of making at least one sale?

In [2]:
# Answer: EXERCISE 1.1.

1-stats.binom.cdf(0,10,0.3 )

0.9717524751

2. Ten coins are tossed simultaneously. Find the probability of getting

i.      at least seven heads

ii.     exactly seven heads

iii.    at most seven heads

In [3]:
# Answer: EXERCISE 1.2.i.
# P(at least seven heads) = P(7T,3Y) + P(8T,2Y) + P(9T,1Y) + P(10T)

stats.binom.pmf(7,10,0.5)+ stats.binom.pmf(8,10,0.5)+ stats.binom.pmf(9,10,0.5)+ stats.binom.pmf(10,10,0.5)

0.17187499999999994

In [47]:
# or alternative answer;

stats.binom.cdf(3,10,0.5)

0.171875

In [31]:
# Answer: EXERCISE 1.2.ii.
# P(exactly seven heads) = P(7T,3Y)

stats.binom.pmf(7,10,0.5)


0.11718749999999999

In [4]:
# Answer: EXERCISE 1.2.iii.
# P(at most seven heads) = 1 - (P(8T,2Y) + P(9T,1Y) + P(10T))

1-(stats.binom.pmf(8,10,0.5)+ stats.binom.pmf(9,10,0.5)+ stats.binom.pmf(10,10,0.5))

0.9453125

In [5]:
# or alternative answer;

stats.binom.cdf(7,10,0.5)

0.9453125

EXERCISE 2. The cycle time for trucks hauling concrete to a highway construction site is uniformly distributed over the interval 50 to 70 minutes. What is the probability that the cycle time exceeds 65 minutes  if it is known that the cycle time exceeds 55 minutes?

Answer: EXERCISE 2.


* In a uniform distribution, the probability is directly proportional to the area of the rectangle;
* CT = Cyle Time 

P(CT>55) = 3/4 = 0.75

P(CT>65) = 1/4 = 0.25

P(CT>65 ∩ CT>55) = P(CT>65)

P(CT>65 | CT>55) = P(CT>65 ∩ CT>55) / P(CT>55) = P(CT>65) / P(CT>55) = 0.25 / 0.75 = 1/3 = 0.3333....=~ 33%

In [6]:
stats.uniform.cdf(70,55,15)-stats.uniform.cdf(65,55,15)

0.33333333333333337

EXERCISE 3. The width of bolts of fabric is normally distributed with mean 950 mm (millimeters) and standard deviation 10 mm.

1. What is the probability that a randomly chosen bolt has a width of between 947 and 958mm?

2. What is the appropriate value for C such that a randomly chosen bolt has a width less than C with probability .8531?

Answer: EXERCISE 3.1.

* Normally distributed, 
mu = 950mm, sigma = 10

P(947< X < 958) = P((947-mu)/sigma < (X-mu)/sigma < (958-mu)/sigma ) = P((947-950)/10 < Z < (958-950)/10) 

P(947< X < 958) = P(-0.3 < Z < 0.8)

P(947< X < 958) = 1 - P(Z<-0.3) - P(Z>0.8) 
                 
P(947< X < 958) = 1 - 0.3821 - 0.2119 (from table z)

P(947< X < 958) = 0.406

Answer: EXERCISE 3.2.

P(Z < C) = 0.8531

P(Z < C) = P((X-mu)/sigma < (C-mu)/sigma) = 0.8531

P(Z < C) = P(Z < (C-950)/10) = 0.8531

(C-950)/10 = 1.05 (from table z)

C -950 = 10.5 

C = 960.5 mm

EXERCISE 4. Suppose scores on exams in statistics are normally distributed with an unknown population mean and a population standard deviation of 3 points. A random sample of 36 scores is taken and gives a sample mean (sample mean score) of 68. Find a confidence interval estimate for the population mean exam score (the mean score on all exams). Find a 90% confidence interval for the true (population) mean of statistics exam scores.

In [14]:
# Answer: EXERCISE 4.

stats.norm.interval(0.90,68,0.5)

(67.17757318652427, 68.82242681347573)

The 90% confidence interval is (67.17757318652427, 68.82242681347573)

EXERCISE 5.  What is the normal body temperature for healthy humans? A random sample of 130 healthy human body temperatures provided by Allen Shoemaker yielded 98.25 degrees and standard deviation 0.73 degrees.

1. Give a 99% confidence interval for the average body temperature of healthy people.

2. Does the confidence interval obtained in part (a) contain the value 98.6 degrees, the accepted average temperature cited by physicians and others? What conclusions can you draw?

In [7]:
# Answer: EXERCISE 5.1.

stats.norm.interval(0.99 ,98.25,0.0640252354)

(98.08508192249006, 98.41491807750994)

Answer: EXERCISE 5.2.

It is outside the range of 98.6 degrees and is in the 0.5% area, so we can claim that the average temperature for healthy people is different from 98.6 degrees.